app/Auth.tsx

```diff
@@ -65,13 +80,36 @@ export default function Auth() {
     }
   };
 
+  useEffect(() => {
+    const retrieveCompostStand = async () => {
+      try {
+        const storedCompostStand = await AsyncStorage.getItem('compostStand');
+        if (storedCompostStand) {
+          const compostStand: CompostStand = storedCompostStand as CompostStand;
+          dispatch(setCompostStand(compostStand));
+        }
+      } catch (error) {
+        console.error('Error retrieving compostStand from local storage:', error);
+      }
+    };
+
+    retrieveCompostStand();
+  }, [dispatch]);
+
+
   const onFailedLogin = () => {
     dispatch(setIsModalVisible(false));
     setRegUI(true);
   };
 
   return (
     <View style={styles.container}>
+      <View>
+        <View style={styles.headerContainer}>
+          <h1 style={styles.headerText}>{i18n.t('main_title')}</h1>
```

Comment:
I think we'll need to run this through our design committee. Please include screenshot in PR but remove text for now 👍

Reply:
Sure :)

<img width="675" alt="Screenshot 2023-11-29 at 16 04 40" src="https://github.com/LiraShapira/app/assets/64899373/b5968a68-391d-4270-b96b-d5045e17e3b1">